avocado/core/varianter.py

```diff
@@ -385,7 +385,7 @@ def _process_default_params(self, args):
         self.default_params.clear()     # We don't need these anymore
         # FIXME: Backward compatibility params, to be removed when 36 LTS is
         # discontinued
-        if (not getattr(args, "variants_skip_defaults", False) and
+        if (not getattr(args, "variants-skip-defaults", False) and
```

Comment:
This confusion was caused by the fact that argparse creates destinations with underscore for options with hyphen. I'd change the destination to `dest="variants_skip_defaults"` instead, but that's not really important and I'm ok with either.